server/server.js

```diff
@@ -12,7 +12,7 @@ require('dotenv').config();
 const app = express();
 const dbURI = `mongodb+srv://jfleming9357:${process.env.MONGO_PASS}@cluster0.v4rli.mongodb.net/papercut?retryWrites=true&w=majority`;
 
-const port = process.env.PORT || 3008;
+const port = process.env.PORT || 3000;
```

Comment:
flip flopping so much

Reply:
I keep trying to set it to 3000 but it gets changed back. I'm just trying to fix the problem I created :(